.github/workflows/PushTemplatesToDB.yml

```diff
@@ -0,0 +1,23 @@
+name: Push Templates to Database
+
+on:
+    push:
+        branches: [ main ]
+    
+jobs:
+  
+  action-PushTemplatesToDB:
+    runs-on: windows-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: '8.0.x'  # or whatever version you're targeting
+          
+      - name: "[Action] Push missing templates to Database"
```

Comment:
Always regenerate STRClient from production swagger.json before! This should increase robustness against service changes. Feel free to call me to talk about this point!